runtime/doc/treesitter.txt

```diff
@@ -822,6 +820,18 @@ get_query_files({lang}, {query_name}, {is_included})
         string[] query_files List of files to load for given query and
         language
 
+get_range({node}, {source}, {metadata})           *vim.treesitter.get_range()*
+    Get the range of a |TSNode|. Can also supply {source} and {metadata} to
```

Comment:
what is `{source}`, conceptually? `{metadata}` is pretty clear from context, but `{source}` is very generic and not a common term in these docs (yet)